tasks/installer.py

```diff
@@ -49,7 +50,9 @@ def build(
     build_type = "-a" if rebuild else ""
     go_build_tags = " ".join(build_tags)
     updater_bin = os.path.join(BIN_PATH, bin_name("installer"))
-    cmd = f"go build -mod={go_mod} {race_opt} {build_type} -tags \"{go_build_tags}\" "
+    static = "CGO_ENABLED=0" if static else ""
```

Comment:
nit: I'd prefer for the flag to reflect that it's about disabling cgo

Reply:
Renamed it `no_cgo` 👍